docs/StardustDocs/topics/apiLevels.md

```diff
@@ -144,17 +149,17 @@ We don't need to interrupt a function call chain and declare a column accessor o
 
 In contrast, generated [extension properties](extensionPropertiesApi.md) are the most convenient and the safest API. 
 Using it, you can always be sure that you work with correct data and types. 
-But its bottleneck is the moment of generation. 
-To get new extension properties you have to run a cell in a notebook, 
+But its bottleneck is the moment of a generation. 
```

Comment:
sentences shouldn't start with "But". I'd say:
"However, its bottleneck is at the moment of generation."